styx-api-service/src/test/java/com/spotify/styx/api/StatusResourceTest.java

```diff
@@ -86,6 +91,12 @@
 
   private Storage storage;
 
+  private static final String AUTH_SERVICE_ACCOUNT = "foo@bar.iam.gserviceaccounts.com";
+  private static final String AUTH_PRINCIPAL = "bar@example.com";
+  private static final ByteString AUTH_PAYLOAD2 = ByteString.encodeUtf8(
```

Comment:
`2`?

Please avoid hand-written json.

Reply:
We have quite some hand-written json in test cases. I think it serves a purpose to ensure data structure changes are caught by test cases, when hand-written plain string gets deserialized into Java object.

Reply:
I think we can talk about and address this issue separately and refactor if we decide to.